### PR TITLE
Upgrade to Elasticsearch 7.3.0 and release version 7.3.0.0

### PR DESCRIPTION
**Pull request by foxdalas**

### CODE REVIEW

**Comment by lukas-vlcek** [comment]
Thanks, I will have a look.
Just FYI, https://github.com/vvanholl/elasticsearch-prometheus-exporter/issues/203 needs to go first.

**Comment by foxdalas** [comment]
@lukas-vlcek https://github.com/vvanholl/elasticsearch-prometheus-exporter/pull/207

**Comment by vvanholl** [comment]
Hi @foxdalas Can you sync from master branch and update release date to today ?
Thanks :)

**Comment by foxdalas** [comment]
@vvanholl np

File: README.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ It collects all relevant metrics and makes them available to Prometheus via the
 
 | Elasticsearch  | Plugin         | Release date |
 | -------------- | -------------- | ------------ |
+| 7.3.0          | 7.3.0.0        | Aug 01, 2019 |
```
Comment (vvanholl) [inline README.md]: Please update to today

Comment (foxdalas) [reply README.md]: @vvanholl fixed

**Comment by vvanholl** [comment]
@foxdalas I have locally tested and it doesn't build correctly. I have the same issues reported here : https://github.com/elastic/elasticsearch/issues/45073

**Comment by vvanholl** [comment]
To help you a bit if you have Docker installed on your system :
```
docker run -it --rm=true -v $(pwd):/home/gradle gradle:5.5.1-jdk12 su gradle -c 'gradle build'
```

**Comment by vvanholl** [comment]
Once solved Gradle <-> ES issue we have new problems due to 7.3.0 upgrade
```
> Task :compileJava
/home/gradle/src/main/java/org/compuscene/metrics/prometheus/PrometheusMetricsCollector.java:27: error: cannot find symbol
import org.elasticsearch.cluster.node.DiscoveryNode.Role;
                                                   ^
  symbol:   class Role
  location: class DiscoveryNode
/home/gradle/src/main/java/org/elasticsearch/action/NodePrometheusMetricsAction.java:24: error: cannot find symbol
public class NodePrometheusMetricsAction extends Action<NodePrometheusMetricsResponse> {
                                                 ^
  symbol: class Action
/home/gradle/src/main/java/org/compuscene/metrics/prometheus/PrometheusMetricsCollector.java:138: error: cannot find symbol
            for (Role r : ns.getNode().getRoles()) {
                 ^
  symbol:   class Role
  location: class PrometheusMetricsCollector
/home/gradle/src/main/java/org/elasticsearch/action/NodePrometheusRequestBuilder.java:29: error: incompatible types: NodePrometheusMetricsAction cannot be converted to ActionType<NodePrometheusMetricsResponse>
        super(client, action, new NodePrometheusMetricsRequest().local(true));
                      ^
/home/gradle/src/main/java/org/elasticsearch/action/NodePrometheusMetricsAction.java:32: error: method does not override or implement a method from a supertype
    @Override
    ^
/home/gradle/src/main/java/org/elasticsearch/action/NodePrometheusMetricsAction.java:38: error: method does not override or implement a method from a supertype
    @Override
    ^
Note: Some messages have been simplified; recompile with -Xdiags:verbose to get full output
6 errors
```

**Comment by foxdalas** [comment]
Yep. I will try to fix it tomorrow

**Pull request closed without merging**